pretyping/pretyping.mli

```diff
@@ -57,6 +57,8 @@ val all_and_fail_flags : inference_flags
 
 val understand_tcc : ?flags:inference_flags -> env -> evar_map ->
   ?expected_type:typing_constraint -> glob_constr -> evar_map * constr
+val understand_tcc_ty : ?flags:inference_flags -> env -> evar_map ->
+  ?expected_type:typing_constraint -> glob_constr -> evar_map * constr * types
```

Comment:
There should at least be a comment explaining that the type ignores the expand_evar flag, although I'd prefer if it wasn't ignored.